api/config-loader.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Config handling - here because of strange babel errors.
+ * By Mia
+ * @author mia-pi-git
+ */
+
+import * as routes from '../config/routes.json';
+import * as fs from 'fs';
+const CONFIG_PATH = 	require.resolve('../config/ls-config');
```

Comment:
Your Babel config apparently can't handle stray whitespace?

```suggestion
const CONFIG_PATH = require.resolve('../config/ls-config');
```